engine/spec/features/error_summary_spec.rb

```diff
@@ -1,24 +0,0 @@
-# frozen_string_literal: true
-
-RSpec.describe "error summary" do
```

Comment:
No explicit error summary test as all steps include some kind of error handling check instead